docs/tutorials/mobile/part-2.md

```diff
@@ -0,0 +1,1169 @@
+---
+title: 'Part 2: Create an Ionic mobile application'
+authors: 'Benjamin Fuentes'
+last_update:
+  date: 12 December 2023
+---
+
+A web3 mobile application is not different from a web2 one in terms of its basic functionality and user interface. Both types of applications can run on smartphones, tablets, and other mobile devices, and both can access the internet and provide various services to users. However, a web3 mobile application differs from a web2 one in terms of its underlying architecture and design principles. A web3 mobile application is built on decentralized technologies, such as blockchain, smart contracts, and peer-to-peer networks, that enable more transparency, security, and autonomy for users and developers.
+
+## Create the Mobile app
+
+Ionic React is a good hybrid solution and compatible with the [BeaconSDK](https://github.com/airgap-it/beacon-sdk) (Typescript). The behavior is equivalent to a classical web development.
+
+> Beacon : the protocol of communication between the dapp and the wallet.
+
+> Note : As of today, it is not recommended to develop a native dapp in Flutter, React Native because or native tools as it requires additional UI works (ex : missing wallet popup mechanism to confirm transactions).
```

Comment:
Not sure if I follow this sentence. Maybe if you removed "because"?

Reply:
yep, I rephrased the sentence and forget to remove it. Good spot